randovania/gui/ui_files/games_tab_planets_zebeth_widget.ui

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>PlanetsZebethGameTabWidget</class>
+ <widget class="QTabWidget" name="PlanetsZebethGameTabWidget">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>574</width>
+    <height>449</height>
+   </rect>
+  </property>
+  <property name="currentIndex">
+   <number>0</number>
+  </property>
+  <widget class="QWidget" name="tab_intro">
+   <attribute name="title">
+    <string>Introduction</string>
+   </attribute>
+   <layout class="QVBoxLayout" name="intro_layout">
+    <item>
+     <layout class="QHBoxLayout" name="intro_cover_layout">
+      <item>
+       <widget class="QLabel" name="game_cover_label">
+        <property name="text">
+         <string>TextLabel</string>
+        </property>
+       </widget>
+      </item>
+      <item>
+       <widget class="QLabel" name="intro_label">
+        <property name="text">
+         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
```

Comment:
Something is very weird with this text
![grafik](https://github.com/randovania/randovania/assets/38186597/7f56b5b8-fb92-4098-bbae-49134181eefe)
For comparison
![grafik](https://github.com/randovania/randovania/assets/38186597/ef7c82c1-4c58-401f-9299-83810b2fcd28)

Reply:
Oops forgot to update the boss keys thing in this tab

Reply:
i was more referring to it being way bolder for some reason.

Reply:
I haven't changed the font

Reply:
well something is clearly different tho :/

Reply:
looked into it, for some reason you have a bunch of extra styles defined in there.
If you're using QT Designer, you can right click the text -> change formatted text. (i think changing the file via a text editor might work too)
you should remove every tag/style that ***isn't*** one of the following
- html
- head
- body
- p
- a href="tab://NAME" (for faq/difference links)
- span style=" text-decoration: underline; color:#007af4;" (for faq/difference links)